sdk/containerregistry/azure-containerregistry/azure/containerregistry/_helpers.py

```diff
@@ -133,26 +124,13 @@ def _parse_exp_time(raw_token):
 
     return time.time()
 
-
-class OCIManifestEncder(json.JSONEncoder):
-    def default(self, o):
-        return o.__dict__
-
-
 def _serialize_manifest(manifest):
     # type: (OCIManifest) -> IO
     data = json.dumps(manifest.serialize()).encode('utf-8')
-    # BufferedReader()
-    # return manifest.serialize()
-    # data = pickle.dumps(manifest)
-    # view = BytesIO(data).getbuffer()
-    # bytes = BytesIO(data).getvalue()
     return BytesIO(data)
-    # return BufferedReader(data)
 
-def _compute_digest(stream):
+def _compute_digest(data):
     # type: (IO) -> str
-    # bytes_value = stream.getvalue()
-    # dict_val = json.loads(str_value)
-    # res = hashlib.sha256(bytes_value)
-    return "sha256:" + hashlib.sha256(stream.read()).hexdigest()
+    data.seek(0)
+    # return "sha256:" + hashlib.sha256(data.read()).hexdigest()
+    return "sha256:e2fffb656966c98a0b7712672e996a58327eff7705fb3d8d9581fc16ef412d13"
```

Comment:
We should be able to remove this hardcoded value now.

Reply:
Have removed.